templates/experiments/chat/components/message_rating.html

```diff
@@ -0,0 +1,20 @@
+<div class="flex flex-row gap-1" id="rating-{{message.id}}">
+    {% if message.rating %}
+        <div class="text-sm">{{ message.rating }}</div>
+    {% else %}
+        <button
+            hx-post="{% url 'experiments:rate_message' team_slug=team_slug message_id=message.id rating='👍' %}"
+            hx-swap="outerHTML"
+            hx-target="#rating-{{message.id}}"
+            class="text-sm p-1 rounded-md hover:bg-gray-50">
+            👍
+        </button>
+        <button
+            hx-post="{% url 'experiments:rate_message' team_slug=team_slug message_id=message.id rating='👎' %}"
+            hx-swap="outerHTML"
+            hx-target="#rating-{{message.id}}"
+            class="text-sm p-1 rounded-md hover:bg-gray-50">
+            👎
+        </button>
```

Comment:
same here but for "bad response"

Reply:
https://github.com/dimagi/open-chat-studio/pull/1004/commits/85d5c5d498a3a2bf97bae7c489300d02d024b17b